data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
> But you can break any server by modifying a script.

what you meant was any tfs right? out there is more gaming servers that use scripting api, for example dota2 server lua api, it's safe enough to valve host any mod developed by community on their own machine, that's a good example of a well done lua api.
have a look at developer.valvesoftware.com
> The amount of damage might differ but you could crash 1.1+ by not knowing how to handle userdata values.

when user turn off script security conversions, server assume he knows what he is doing, btw i think this could be changed, out of topic tho, nothing going to arg it here.

> You can also break 0.x servers via Lua scripts.

idk why you are referring to unsupported and legacy version, if you want my opnion 0.x was horrible, bad design and kludges over whole code.

>The point is to make people learn, we cant make everything noob proof without having a billion checks for everything the server handles.

it's not being 'noob proof' but beeing stable.

@conde2 

> every time someone answers here all participants are notified.

sorry for the inconvenience, but it may be discussed here imo, it's about project, to avoid this kind of discussion in futher i think someone with merge access should write a contribution guide, btw there is a left panel that you can always unsubscribe from PRs you're not interested in.